wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -634,10 +659,526 @@ public boolean testWUQuery() throws Exception
             return false;
         }
     }
+    
+    /**
+     * Converts EspException from v1_56 to v1_58
+     * 
+     * @param org.hpccsystems.ws.client.gen.wsworkunits.v1_56.EspException in
+     * @return EspException
+     */
+    private EspException espexceptionConvert(org.hpccsystems.ws.client.gen.wsworkunits.v1_56.EspException in)
+    {
+    	EspException out = new EspException();
+    	if(in.getCode() != null) out.setCode(in.getCode());
+    	if(in.getAudience() != null) out.setAudience(in.getAudience());
+    	if(in.getSource() != null) out.setSource(in.getSource());
+    	if(in.getMessage() != null) out.setMessage(in.getMessage());
+    	return out;
+    }
+    
+    /**
+     * Converts ArrayOfEspException from v1_56 to v1_58
+     * 
+     * @param org.hpccsystems.ws.client.gen.wsworkunits.v1_56.ArrayOfEspException in
+     * @return ArrayOfEspException
+     */
+    private ArrayOfEspException arrayofespexceptionsConvert(org.hpccsystems.ws.client.gen.wsworkunits.v1_56.ArrayOfEspException in)
+    {
+    	ArrayOfEspException out = new ArrayOfEspException();
+    	if(in.getSource() != null) out.setSource(in.getSource());
+    	if(in.getException() != null)
+    	{
+    		EspException[] temp_exceptions = new EspException[in.getException().length];
+    		for(int i = 0; i < in.getException().length; i++)
+    		{
+    			temp_exceptions[i] = espexceptionConvert(in.getException(i));
+    		}
```

Comment:
Make sure you're not introducing tabs, make use of the formatter for Eclipse included at the root of the git japis repo